frontend/src/components/StatusTags.vue

```diff
@@ -18,22 +18,25 @@ SPDX-License-Identifier: Apache-2.0
       </status-tag>
     </div>
     <template v-if="showStatusText">
-      <div v-for="({ description }) in errorCodeObjects" :key="description" class="mt-1">
-        <div class="font-weight-bold error--text wrap" v-html="description" />
+      <div v-for="({ description, hint }) in errorCodeObjects" :key="description" class="mt-1">
+        <div class="font-weight-bold error--text wrap-text" v-html="description" />
```

Comment:
We still have `v-html`?

Reply:
IDK how this could happen I thought I removed it.